controller/lib/core/blower_pid.cpp

```diff
@@ -27,28 +27,17 @@ static double Input;
 static double Output;
 
 // PID-tuning were chosen by following the Ziegler-Nichols method,
-// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method.
-//
-// Here's an achieved pressure vs. desired pressure trace collected on Alpha
-// build, 2020-05-01, with the following settings.
-//
-//   PEEP = 10 cm H2O,
-//   PIP = 30 cm H2O,
-//   RR = 10 breaths/min
-//   I:E = 2
-//
-// https://drive.google.com/file/d/13QyvO1ptUYYUAs8qT0dCyW9f1XZ8w7iw
+// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method
```

Comment:
Beware that @asmodai27 's #221 is changing the PID controller behavior a bit, he says this may need to be re-tuned.